cmd/tools/commentMonitor/main.go

```diff
@@ -0,0 +1,144 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+
+	"github.com/google/go-github/v26/github"
+	"golang.org/x/oauth2"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+const prombenchURL = "http://prombench.prometheus.io"
+
+var regex string
+var input string
+var output string
+
+func writeArgs(arglist []string) {
+	for i, arg := range arglist[1:] {
+		data := []byte(arg)
+		filename := fmt.Sprintf("ARG_%d", i)
+		err := ioutil.WriteFile(filepath.Join(output, filename), data, 0644)
+		log.Printf(filepath.Join(output, filename))
+		if err != nil {
+			log.Fatalln(err)
+		}
+	}
+}
+
+func memberValidation(authorAssociation string) error {
+	if (authorAssociation != "COLLABORATOR") && (authorAssociation != "MEMBER") {
+		return fmt.Errorf("not a member or collaborator")
+	}
+	return nil
+}
+
+func regexValidation(regex string, comment string) ([]string, error) {
+	argRe := regexp.MustCompile(regex)
+	if argRe.MatchString(comment) {
+		arglist := argRe.FindStringSubmatch(comment)
+		return arglist, nil
+	}
+	return []string{}, fmt.Errorf("invalid command")
+}
+
+func postComment(client *github.Client, owner string, repo string, prnumber int, comment string) error {
+	issueComment := &github.IssueComment{Body: github.String(comment)}
+	issueComment, _, err := client.Issues.CreateComment(context.Background(), owner, repo, prnumber, issueComment)
+	return err
+}
+
+func main() {
+	app := kingpin.New(filepath.Base(os.Args[0]), "simpleargs github comment extract")
+	app.Flag("input", "path to event.json").Default("/github/workflow/event.json").StringVar(&input)
+	app.Flag("output", "path to write args to").Default("/github/home").StringVar(&output)
+	app.Arg("regex", "Regex pattern to match").Required().StringVar(&regex)
+	app.HelpFlag.Short('h')
+	kingpin.MustParse(app.Parse(os.Args[1:]))
+
+	// Github client for posting comments.
+	token := os.Getenv("GITHUB_TOKEN")
+	ctx := context.Background()
+	ts := oauth2.StaticTokenSource(
+		&oauth2.Token{AccessToken: token},
+	)
+	tc := oauth2.NewClient(ctx, ts)
+	client := github.NewClient(tc)
+
+	// Reading event.json.
+	os.MkdirAll(output, os.ModePerm)
+	data, err := ioutil.ReadFile(input)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	// Parsing event.json.
+	event, err := github.ParseWebHook("issue_comment", data)
+	if err != nil {
+		log.Fatalln("could not parse = %v\n", err)
```

Comment:
Why `\n` isn't this added automatically?
```suggestion
		log.Fatalln("could not parse = %v", err)
```